l3kernel/doc/source3body.tex

```diff
@@ -335,6 +335,21 @@ \section{Naming functions and variables}
     lists (with access at both ends) and stacks.
 \end{description}
 
+\subsection{Behavior of \texttt{c}-type expansion where \texttt{N}-type token
+  resulting from expansion is undefined}
+
+When \texttt{c}-type expansion is applied, it will produce an \texttt{N}-type
+token to be consumed by the underlying function. If the result of this process
```

Comment:
"underlying" needed?

Reply:
I think so - I'm trying to get at `\foo:c` => `\foo:N`